app/serializers/costs_parameters_serializer.rb

```diff
@@ -211,7 +211,8 @@ def node_rows_for_subgroup(graph, group, subgroup)
         subgroup, # Subgroup
         node.key, # Key
         format_num(node.query.total_costs_per(:node) - 
-          node.query.fuel_costs_per(:node) - node.query.co2_emissions_costs_per(:node)), # Total costs (eur)
+          node.query.fuel_costs_per(:node) - node.query.co2_emissions_costs_per(:node) - 
+          node.query.captured_biogenic_co2_costs_per(:node)), # Total costs (eur)
```

Comment:
This method still produces an error when downloading the costs export. It seems that the AREA attribute `captured_biogenic_co2_price` is not found in the molecule graph, which is strange, because the AREA attribute `co2_price` works generally the same way.